modules/compute-vm/variables.tf

```diff
@@ -187,14 +187,22 @@ variable "network_interfaces" {
 variable "options" {
   description = "Instance options."
   type = object({
-    allow_stopping_for_update = bool
-    deletion_protection       = bool
-    spot                      = bool
+    allow_stopping_for_update        = bool
+    deletion_protection              = bool
+    spot                             = bool
+    spot_instance_termination_action = string
```

Comment:
can we rename to `instance_termination_action` like in the resource, or just `termination_action`?